tests/recipes/test_pipeline.py

```diff
@@ -143,7 +143,7 @@ def test_recipes_log_to_expected_mlflow_backend_with_expected_run_tags_once_on_r
     recipe.run()
 
     logged_runs = mlflow.search_runs(experiment_names=[experiment_name], output_format="list")
-    assert len(logged_runs) == 1
+    number_of_logged_runs = len(logged_runs)
```

Comment:
Because of autoML change in recipes/example, the number of runs are not guarantee to be 1. So we are testing that number of runs created in the first try is to number of runs created if we re-run the pipeline